contrib/marketplace/src/App/MarketplaceList.tsx

```diff
@@ -0,0 +1,410 @@
+// Copyright (c) Microsoft Corporation
+// All rights reserved.
+//
+// MIT License
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+// documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+// the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+// to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+// BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import React from "react";
+import {
+  DocumentCard, DocumentCardActions, DocumentCardActivity, DocumentCardLogo, DocumentCardStatus, DocumentCardTitle,
+  DefaultPalette, Fabric, Icon, IconButton, Label, Persona, PersonaSize, Stack, Spinner, SpinnerSize, Text, TextField,
+  initializeIcons, mergeStyleSets,
+} from "office-ui-fabric-react";
+import classNames from "classnames/bind";
+import yaml from "yaml";
```

Comment:
nodeca's [js-yaml](https://github.com/nodeca/js-yaml) is more popular.
Have you tried it?

Reply:
`js-yaml`'s browser library cannot be used directly in nodejs, so @Gerhut suggested to use `yaml` which is a pure browser library for YAML.

Reply:
All right

Reply:
updated